.github/dependabot.yaml

```
@@ -1,7 +1,7 @@
 version: 2
 updates:
 - package-ecosystem: cargo
-  directory: "/"
+  directory: "/trusted"
   schedule:
     interval: daily
   open-pull-requests-limit: 10
```

Dependabot can also notify us of GHA updates 
```suggestion
  open-pull-requests-limit: 10
- package-ecosystem: "github-actions"
  directory: "/"
  schedule:
    # Check for updates to GitHub Actions every weekday
    interval: "daily"
```